jinahub/segmenters/Sentencizer/sentencizer.py

```diff
@@ -35,6 +36,8 @@ def __init__(
         :param punct_chars: the punctuation characters to split on,
             whatever is in the list will be used,
             for example ['!', '.', '?'] will use '!', '.' and '?'
+            If smart_tokenizer=True is passed to segment, punct_chars is
```

Comment:
I think it would be good to have an `__init__` parameter that indeed can be overriden by `parameters`

Reply:
Hi, sorry but I did not well understand, do you mean to add 'smart_tokenizer' param in the class __init__?